src/index.html

```diff
@@ -18,5 +22,94 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
```

Comment:
For the `href` attribute in the <a> tags, ensure there are no spaces and that it starts with a `#` symbol. For example, change `href="samsung"` to `href="#samsung"`.